pandas/core/indexes/base.py

```diff
@@ -1921,7 +1921,7 @@ def sortlevel(
         level=None,
         ascending: bool | list[bool] = True,
         sort_remaining=None,
-        na_position: str_t = "first",
+        na_position: Literal["first", "last"] = "first",
```

Comment:
Could use `na_position: NaPosition = "first",`
https://github.com/pandas-dev/pandas/blob/c85bbc6913b9c4d2acc53c54cc858c84bf0658d1/pandas/_typing.py#L362

Reply:
Thanks for the review. This makes a lot of sense. Addressed in b1ef125

Reply:
Hmm got a strange error in Minimum Version check.

```
2023-04-19T05:22:59.5815494Z error    libmamba Bad conversion of configurable 'remote_connect_timeout_secs' from source '~/.condarc' : yaml-cpp: error at line 14, column 30: bad conversion
```

This doesn't seem to be related to the change?